docking.js

```diff
@@ -1096,6 +1105,24 @@ var DockedDash = GObject.registerClass({
                 this.remove_style_class_name('extended');
             }
         }
+        this._updateVisibleDesktop();
+    }
+
+    _updateVisibleDesktop() {
+        if (!DockManager.settings.get_boolean('intellihide'))
+            return;
+        let monitor = this._monitorIndex;
+        if (this._isHorizontal) {
+            if (this._position == St.Side.BOTTOM)
+                Me.imports.extension.DesktopIconsUsableArea.setMargins(monitor, 0, this._box.height, 0, 0);
```

Comment:
Ad a `const` definition for this since we're reusing it multiple times

Reply:
Done.

Reply:
But not exactly as you probably expected, because I'm accessing the actual object, not the file, so I can't define a CONST at the beginning, because the object changes each time the extension is disabled and enabled again, and also the code must wait until the object is created to be able to use it.